examples/sqlite/src/main.rs

```diff
@@ -1,3 +1,8 @@
+//! Run with
```

Comment:
I believe since this is a separate project now we can divide this file into a few smaller ones to make it less scary ;).

Reply:
I've broken this up a bit. There's more we could do with this, but maybe this is good enough to iterate on (as in we can keep building on this incrementally). Let me know if you hate it. 🙂